app/models/user.rb

```diff
@@ -278,7 +278,7 @@ def questions
     Node.questions.where(status: 1, uid: id)
   end
 
-  def content_followed_in_past_period(time_period)
+  def self.content_followed_in_past_period(time_period)
```

Comment:
Ah, here, actualy perhaps you want to skip the `.self` -- so you can run this for a specific user. I'll elaborate in the controller code.